enterprise/settings/test.py

```diff
@@ -148,8 +148,8 @@ def root(*args):
 
 DEFAULT_FROM_EMAIL = 'course_staff@example.com'
 
-USER_THROTTLE_RATE = '90/minute'
-SERVICE_USER_THROTTLE_RATE = '100/minute'
+USER_THROTTLE_RATE = '190/minute'
```

Comment:
Now, `test` user (used for the client) has the same `id` every time so 90 throttle rate is not enough for all tests.